app/models/user.rb

```diff
@@ -38,6 +35,15 @@ class User < ActiveRecord::Base
   PROVIDER_SIDEWALKS = 'sidewalks'
   PROVIDER_TWITTER = 'twitter'
 
+  scope :where_needs_triage, -> (params = {}) do
```

Comment:
Unused block argument - `params`. If it's necessary, use `_` or `_params` as an argument name to indicate that it won't be used. Also consider using a proc without arguments instead of a lambda if you want it to accept any arguments but don't care about them.